CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Lethe
+
+Lethe is an open source and we try to follow the deal.II mentality of having it
+developed an open accessible environment. Consequently, all scientific
+development made within Lethe are accessible to everyone even as they are
+currently being written. Anybody that wishes to use and contribute to a feature
+is welcomed to do so.
+
+This document aims at establishing from guidelines for contributions within
+Lethe.
+
+
+# How can I contribute?
+
+Contributions through code or documentation should be done through pull
+request at the official Github repository of Lethe : https://github.com/lethe-cfd/lethe.
+We recommend that users either request access to the Lethe repository or
+create their own fork of Lethe on their own github account. This can then be
+used to open pull requests
+
+# What should be the content of a pull request (PR)?
+
+A pull request should contain the following elements:
+
+- A brief title (less than 60 characters) that describes the goal of the pull
+requests
+- A detailed description of the content added by the pull request:
+i) If the PR adds a new feature. The feature should be documented, tests (with
+  a unit tests and/or an application test) and if the features adds or alters
```

Comment:
I'd remove the "s" from feature, or change the uses of the word in the rest of the paragraph.